docs_src/microservices/device/Ch-DeviceServiceList.md

```diff
@@ -0,0 +1,32 @@
+# Device Service Support
+
+The following table lists the EdgeX device services and protocols they support.
+
+| Device Service Repository | Protocol | Releases | Status | Comments |
+|----------------------------------------------------------------------|------------|-------------|----------|----------------|
+| [device-camera-go]( https://github.com/edgexfoundry/device-camera-go)| ONVIF | Delhi-Jakarta|Active| Not a full ONVIF implementation, but a good starter|
+| [device-rest-go]( https://github.com/edgexfoundry/device-rest-go) | REST | Edinburgh-Jakarta| Active| provides one-way communications only.  Allows posting of binary and JSON data via REST.  Events are single reading only.|
+| [device-rfid-llrp-go]( https://github.com/edgexfoundry/device-rfid-llrp-go) | LLRP | Jakarta| Active| Communications with RFID readers via LLRP |
```

Comment:
For now this should just be `1.x` since don't see the V2 migration happening prior Jakarta